simplefs/simplefs.go

```diff
@@ -64,6 +64,7 @@ var errOnlyRemotePathSupported = simpleFSError{"Only remote paths are supported
 var errInvalidRemotePath = simpleFSError{"Invalid remote path"}
 var errNoSuchHandle = simpleFSError{"No such handle"}
 var errNoResult = simpleFSError{"Async result not found"}
+var errInvalidPathType = simpleFSError{"Invalid path type"}
```

Comment:
This is useful, but please use it in `getFS()` as well.

Reply:
I guess you don't need this anymore?

Reply:
Nope. And somehow I missed most of the comments here when I fixed stuff up  so I never saw this comment in the first place.. whoops